app.rb

```diff
@@ -0,0 +1,70 @@
+require_relative 'person'
+require_relative 'student'
+require_relative 'teacher'
+require_relative 'rental'
+require_relative 'book'
+class App
+  attr_accessor :books, :peoples, :rentals
+
+  def initialize
+    @books = []
+    @peoples = []
+    @rentals = []
+  end
+
+  def list_books
+    if @books.empty?
+      puts 'No books available'
+    else
+      @books.each_with_index do |book, i|
+        puts "Number: #{i + 1}, Title: #{book.title}, Author: #{book.author}"
+      end
+    end
+  end
+
+  def list_people
+    if @peoples.empty?
+      puts 'There are no people'
+    else
+      @peoples.each_with_index do |person, i|
+        puts "Number: #{i + 1}, Name: #{person.name}, age: #{person.age}, ID: #{person.id}"
```

Comment:
- [ ] Kindly fix the UI where you list people created. There's an error where `name` it's equal to the `parent_permission` and not to the name of the person created

![image](https://github.com/rbd3/School-Library/assets/93630700/9dd0c86d-31cb-43e3-911f-4d49a4f35c8e)